addons/medical_status/XEH_preInit.sqf

```diff
@@ -26,18 +26,31 @@ PREP_RECOMPILE_END;
 addMissionEventHandler ["EntityKilled", {_this call FUNC(handleKilledMission)}];
 
 if (hasInterface) then {
-    //Add Inventory action to uncon units
+    //Add inventory and open backpack actions to uncon units
     ["CAManBase", "init", {
         params ["_unit"];
 
         private _id = _unit addAction ["", {
-            params ["_target", "_caller", "_actionId", "_arguments"];
+            params ["_target", "_caller"];
 
             _caller action ["Gear", _target];
-        }, nil, 5.1, true, true, "gear", "_target getVariable ['ACE_isUnconscious',false] && {alive _target}", 3.5];
+        }, nil, 5.1, true, true, "gear", toString {
+            (_target isNotEqualTo ACE_player) &&
+            {(lifeState _target) isEqualTo "INCAPACITATED"}
+        }, 2];
 
         _unit setUserActionText [_id, localize "STR_ACTION_GEAR", "<img image='\A3\ui_f\data\igui\cfg\actions\gear_ca.paa' size='2.5' shadow=2 />"];
 
+        _unit addAction ["OpenBag", {
+            params ["_target", "_caller"];
+
+            _caller action ["OpenBag", _target];
+        }, nil, 5.2, true, true, "", toString {
+            (_target isNotEqualTo ACE_player) &&
+            {!((lifeState _target) in ["HEALTHY", "INJURED", "INCAPACITATED"])} &&
+            {!(isNull (backpackContainer _target)) && {!((backpackContainer _target)) isKindOf "Weapon_Bag_Base"}} &&
```

Comment:
```suggestion
            {private _backpack = backpackContainer _target; !isNull _backpack && {!isClass (configOf _backpack / "assembleInfo")}} &&
```

fix  `Error !: Type Object, expected Bool`
this might be a little safer if someone has assemble configs without inheriting

Reply:
This might be a bit more universal:
```SQF
private _backpackContainer = backpackContainer _target;
private _backpackConfig = configOf _backpackContainer;

(_target isNotEqualTo ACE_player) &&
{!((lifeState _target) in ["HEALTHY", "INJURED", "INCAPACITATED"])} &&
{!isNull _backpackContainer} &&
{!lockedInventory _backpackContainer} &&
{maxLoad _backpackContainer > 0} &&
{getNumber (_backpackConfig >> "disableInventory") != 1} &&
{_target setUserActionText [_actionId, format [localize "STR_ACTION_OPEN_BAG", getText (_backpackConfig >> "displayName")]]; true}
```